coil-core/src/jvmCommonMain/kotlin/coil3/decode/ImageSource.kt

```diff
@@ -0,0 +1,12 @@
+package coil3.decode
+
+import coil3.annotation.ExperimentalCoilApi
+import java.nio.ByteBuffer
+
+/**
+ * Metadata containing the [bytebuffer] of a ByteBuffer, maybe direct
+ */
+@ExperimentalCoilApi
+class ByteBufferMetadata(
```

Comment:
I think this file might have to be renamed to `ImageSource.jvm.kt` to avoid class name conflicts.

Reply:
Well, it doesn't have top-level functions/field, so will not have clash.